### PR TITLE
Improving test coverage

### PR DESCRIPTION
**Pull request by jain651**
ref #272 
removing some mooseError that cannot execute.

### CODE REVIEW

File: test/tests/concrete_moisture_heat_transfer/tests [hunk 1]
```diff
@@ -9,7 +9,7 @@
     rel_err = 1e-6
     abs_zero = 1e-10
   []
-  issues = '#245'
```
Comment (bwspenc) [inline test/tests/concrete_moisture_heat_transfer/tests:12]: I'd just keep issue 245 here. The 272 issue isn't really relevant.

File: test/tests/concrete_moisture_heat_transfer/tests [hunk 2]
```diff
@@ -35,11 +35,13 @@
...
-                 'Materials/concrete/n=16 '
-                 'Materials/concrete/critical_relative_humidity=0.75 '
-                 'Materials/concrete/coupled_moisture_diffusivity_factor=1e-5 '
+                 'Materials/concrete/moisture_model=Mensi '
+                 'Materials/concrete/A=1.042e-13 '
+                 'Materials/concrete/B=0.05 '
+                 'Materials/concrete/cement_mass=354 '
+                 'Materials/concrete/water_to_cement_ratio=0.5 '
+                 'Tmin=10 '
+                 'Tmax=800 '
```
Comment (bwspenc) [inline test/tests/concrete_moisture_heat_transfer/tests:44]: What do Tmin and Tmax do? I see that 'Tmax' shows up in the input file at the top level, but it's unclear to me what it's used for.

File: test/tests/concrete_moisture_heat_transfer/tests [hunk 3]
```diff
@@ -602,5 +608,68 @@
...
+      expect_err = "Temperature outside of the range for the KODUR-2004 thermal model"
+      cli_args = 'Materials/concrete/thermal_model=KODUR-2004 '
+                'Materials/concrete/aggregate_type=Siliceous '
+                'Materials/concrete/moisture_model=Mensi '
+                'Materials/concrete/A=1.042e-13 '
+                'Materials/concrete/B=0.05 '
+                'Materials/concrete/cement_mass=354 '
+                'Materials/concrete/water_to_cement_ratio=0.5 '
+                'Tmax=650 '
+      allow_test_objects = True
```
Comment (bwspenc) [inline test/tests/concrete_moisture_heat_transfer/tests:623]: Why do we need to allow test objects? I see that parameter set for the other existing error tests, but I think it may not be needed. If not, please remove it from all the tests.

File: src/materials/ConcreteThermalMoisture.C [hunk 4]
```diff
@@ -386,8 +386,6 @@ ConcreteThermalMoisture::computeProperties()
               _thermal_conductivity[qp] = 1.4875;
             else if (T >= 20.0 && T < 800.0)
```
Comment (bwspenc) [inline src/materials/ConcreteThermalMoisture.C:387]: You probably don't need the T<800 check here, right?

**Review by bwspenc — CHANGES_REQUESTED** [review]

File: test/tests/concrete_moisture_heat_transfer/tests (hunk 3 above)
Comment (jain651) [reply test/tests/concrete_moisture_heat_transfer/tests:623]: Tmax and Tmin are the parameters used to change the temperature range in the boundary conditions so that we cover all the lines in the if-else statement for each model. They are needed as I am using these parameters in the BC's xy_data.

File: test/tests/concrete_moisture_heat_transfer/tests (hunk 2 above)
Comment (jain651) [reply test/tests/concrete_moisture_heat_transfer/tests:44]: Tmax and Tmin are the parameters used to change the temperature range in the boundary conditions so that we cover all the lines in the if-else statement for each model. They are needed as I am using these parameters in the BC's xy_data.

File: test/tests/concrete_moisture_heat_transfer/tests (hunk 3 above)
Comment (jain651) [reply test/tests/concrete_moisture_heat_transfer/tests:623]: I have removed it now from all test cases.

File: test/tests/concrete_moisture_heat_transfer/tests (hunk 1 above)
Comment (jain651) [reply test/tests/concrete_moisture_heat_transfer/tests:12]: I have modified it now.

File: src/materials/ConcreteThermalMoisture.C (hunk 4 above)
Comment (jain651) [reply src/materials/ConcreteThermalMoisture.C:387]: I have removed it now.

File: test/tests/concrete_moisture_heat_transfer/tests (hunk 2 above)
Comment (bwspenc) [reply test/tests/concrete_moisture_heat_transfer/tests:44]: That makes sense, but I'm still missing how they are used in the xy_data.

**Review by bwspenc — APPROVED** [review]
